apps/coscrad-frontend/src/components/nav-menu/nav-menu-presenter.tsx

```diff
@@ -0,0 +1,58 @@
+import MenuRoundedIcon from '@mui/icons-material/MenuRounded';
+import { Box, IconButton, Menu } from '@mui/material';
+import { useState } from 'react';
+import AuthenticationButton from '../authentication-button/authentication-button';
+import { NavItemInfo } from './nav-menu-container';
+import { NavMenuItem } from './nav-menu-item';
+
+interface NavMenuPresenterProps {
+    navItemInfos: NavItemInfo[];
+}
+
+export const NavMenuPresenter = ({ navItemInfos }: NavMenuPresenterProps): JSX.Element => {
+    const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+    const open = Boolean(anchorEl);
```

Comment:
I'm not a fan of this, although at least it makes the casting explicit. Also, boolean vars should start with "is", "does", "had", or "should". How about
```ts
const isOpen = !isNull(anchorEl) // import isNull from @coscrad/validation-constraints
```

Reply:
Done